docker-compose.yml

```diff
@@ -15,7 +16,7 @@ services:
     networks:
       - backend_nt
     healthcheck:
-      test: ["CMD", "redis-cli", "ping"]
+      test: ["CMD", "redis-cli", "-p", "${REDIS_PORT}", "ping"]
```

Comment:
Port wasn't already specified in service configuration?

Reply:
if port not specified healthcheck doesnt work on other port.
For example if REDIS is launch with PORT 6380 healthcheck will not pass